### PR TITLE
Add documentation on notification presets/format

### PR DESCRIPTION
**Pull request by Dalje-et**
### What does this PR do?

Adds documentation on what content can be toggled on/off in monitor notifications.

### Motivation
Feature that needs documentation.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/notify/_index.md [hunk 1]
```diff
@@ -114,6 +114,17 @@ Disk space is low @ops-team@company.com
 
 {{% notifications-integrations %}}
 
+### Toggle additional content
+
+Providing additional context, monitor notifications can include content such as the monitor's query, the @-mentions used, metric snapshots (for metric monitors), and links back to relevant pages in Datadog. However, you might decide that for some monitors this additional content is not needed. For that, you can decide on each inidividual monitor, which content you would like to strip away from its notifications. The options are:
```
Comment (cswatt) [inline content/en/monitors/notify/_index.md:119]:
```suggestion
Monitor notifications can provide additional context, such as the monitor's query, the @-mentions used, metric snapshots (for metric monitors), and links back to relevant pages in Datadog. You can configure the types of additional context included in each monitor's notifications. The options are:
```

File: content/en/monitors/notify/_index.md [hunk 2]
```diff
@@ -114,6 +114,17 @@ Disk space is low @ops-team@company.com
...
+
+- **Default**: no content is hidden
+- **Hide Query**: removes the monitor's query from its notification
+- **Hide Handles**: strips away the @-mentions that are used in the notification message
+- **Hide All**: notification does not include query, handles, any snapshot (for metric monitors), and additional links in footers
```
Comment (cswatt) [inline content/en/monitors/notify/_index.md:124]:
```suggestion
- **Default**: No content is hidden.
- **Hide Query**: Remove the monitor's query from the notification message.
- **Hide Handles**: Remove the @-mentions that are used in the notification message.
- **Hide All**: Notification message does not include query, handles, any snapshots (for metric monitors), or additional links in footers.
```

grammar stuff

File: content/en/monitors/notify/_index.md [hunk 3]
```diff
@@ -114,6 +114,17 @@ Disk space is low @ops-team@company.com
...
+
+Note, some content might not be displayed by defauly depending on the integration.
```
Comment (cswatt) [inline content/en/monitors/notify/_index.md:126]:
```suggestion
**Note**: Depending on the integration, some content may not be displayed by default.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Comment by Dalje-et** [comment]
Hey @cswatt, do you think this one is good to go? :)

File: content/en/monitors/notify/_index.md [hunk 4]
```diff
@@ -114,6 +114,21 @@ Disk space is low @ops-team@company.com
 
 {{% notifications-integrations %}}
 
+### Toggle additional content
+
+Providing additional context, monitor notifications can include content such as the monitor's query, the @-mentions used, metric snapshots (for metric monitors), and links back to relevant pages in Datadog. However, you might decide that for some monitors this additional content is not needed. For that, you can decide on each individual monitor, which content you would like to strip away from its notifications.
+
+{{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:70%;" >}}
```
Comment (estherk15) [inline content/en/monitors/notify/_index.md:121]:
```suggestion
{{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:100%;" >}}
```

File: content/en/monitors/notify/_index.md [hunk 5]
```diff
@@ -114,6 +114,21 @@ Disk space is low @ops-team@company.com
 
 {{% notifications-integrations %}}
 
+### Toggle additional content
+
+Monitor notifications include content such as the monitor's query, the @-mentions used, metric snapshots (for metric monitors), and links back to relevant pages in Datadog. You have the option to choose which content you would like to include or exclude from notifications for individual monitors. 
+
+{{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:100%;" >}}
```
Comment (estherk15) [inline content/en/monitors/notify/_index.md:121]:
```suggestion
{{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:70%;" >}}
```

**Review by estherk15 — APPROVED** [review]

**Pull request merged**